dev-util/sysprof/sysprof-48.0.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils greadme meson systemd xdg
+
+DESCRIPTION="System-wide Linux Profiler"
+HOMEPAGE="https://www.sysprof.com/"
+
+LICENSE="GPL-3+ GPL-2+"
+API_VERSION="4"
+SLOT="0/${API_VERSION}"
+KEYWORDS="~amd64 ~arm64 ~loong ~x86"
+IUSE="gtk systemd test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
```

Comment:
I suspect this is missing an elfutils dependency? See https://gitlab.gnome.org/GNOME/sysprof/-/commit/1bd79af439eb4eb2ffbac7414b2a26a4bfa11ad3.

Reply:
It's pretty common to not have elfutils on your system until a kernel build needs it for a minimal chroot, e.g. it happens a lot with musl.

As for the `@system` thing, you should treat that more in terms of "RDEPEND on sed is silly", it doesn't really apply for libraries.